test/phpunit.xml.dist

```diff
@@ -1,33 +0,0 @@
-<phpunit 
```

Comment:
Why did you rmeove the phpunit.xml.dist file? The nice part about is that it can be locally overwritten and the .dist stays still in place.